From 369e075aff64c16f4ed0a764b585e95ab3c5364b Mon Sep 17 00:00:00 2001
From: Guochun Huang <hero.huang@rock-chips.com>
Date: Mon, 2 Dec 2019 16:17:26 +0800
Subject: [PATCH] media: Add MEDIA_BUS_FMT_RGB101010_1X7X5_SPWG/JEIDA media bus
 code definitions

This patch adds a new RGB media bus formats that describe
30-bit samples transferred over an LVDS bus with five
differential data pairs, serialized into 7 time slots,
using standard VESA/JEIDA data ordering.

Change-Id: I922a48c70f96727f0292004bb38dabab99f6a918
Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
---
 include/uapi/linux/media-bus-format.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/uapi/linux/media-bus-format.h b/include/uapi/linux/media-bus-format.h
index be55ae7d58fc..4ccd02049c3e 100644
--- a/include/uapi/linux/media-bus-format.h
+++ b/include/uapi/linux/media-bus-format.h
@@ -34,7 +34,7 @@
 
 #define MEDIA_BUS_FMT_FIXED			0x0001
 
-/* RGB - next is	0x1022 */
+/* RGB - next is	0x1024 */
 #define MEDIA_BUS_FMT_RGB444_1X12		0x1016
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_BE	0x1001
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_LE	0x1002
@@ -68,6 +68,8 @@
 #define MEDIA_BUS_FMT_SRGB888_DUMMY_4X8		0x101f
 #define MEDIA_BUS_FMT_SBGR888_DUMMY_4X8		0x1020
 #define MEDIA_BUS_FMT_SRBG888_DUMMY_4X8		0x1021
+#define MEDIA_BUS_FMT_RGB101010_1X7X5_SPWG      0x1022
+#define MEDIA_BUS_FMT_RGB101010_1X7X5_JEIDA     0x1023
 
 /* YUV (including grey) - next is	0x202d */
 #define MEDIA_BUS_FMT_Y8_1X8			0x2001
-- 
2.35.3

